models/users.ts

```diff
@@ -1,6 +1,7 @@
 export interface UserDoc {
   name: string
   email: string
+  organization: string
```

Comment:
Should be optional

Reply:
It is I just pass an empty string into this if the UI doesn't receive input for organization, I just thought it would be the same thing since "" (empty string) is false and an undefined property is also false

Reply:
Imo it's more clear to have `organization` as an undefined property so someone looking at the type knows it's optional rather than having it turn out to be an empty string